configure.py

```diff
@@ -171,15 +173,23 @@ def identify_best_standard(cpp_standards, compiler):
 
 MODE_TO_CMAKE_BUILD_TYPE = {'release': 'RelWithDebInfo', 'debug': 'Debug', 'dev': 'Dev', 'sanitize': 'Sanitize' }
 
+SANITIZER_TO_CMAKE = {'address': 'address', 'undefined': 'undefined_behavior'}
```

Comment:
why not just use `undefined_behavior`? what's the merit of introducing another layer?

Reply:
Agree, the same thing happens in `FindSanitizers.cmake`. I deleted it